### PR TITLE
Fix for TextEditor (iOS 15)

### PR DESCRIPTION
**Pull request by sapieneptus**
### Description

I noticed that locally [^1] `introspectTextView` was not finding the `UITextView` inside of a `TextEditor`.  I was able to reproduce this in upstream `master` by updating the iOS target to 15.0 for the tests, so presumably Apple made some change to `TextEditor` recently.

I was able to resolve this issue by changing the selector for `introspectTextView` from `siblingContaining` to ~`siblingContainingOrAncestor`, under the assumption that it's behaving similarly to recent versions of `ScrollView` (which use the same selector).~

_Edit:_ After following the READMEs [suggestion](siblingContainingOrAncestorOrAncestorChild) to try adding an extension locally (since I'm not sure how long this will take to get merged), it now seems that it only works with the `siblingContainingOrAncestorOrAncestorChild` selector 🤔 

### CI is Failing...

... because it's using a simulator too old to run iOS 15.0. I'm not entirely sure what's preferred in this case. We could revert my changes to the build target, but that would in effect hide the problem that this PR is solving. 

### Disclaimer

The tests now pass with the selector changed, but I'm not in a position to say that this is necessarily the best solution, as it is more or less a guess based on the fact that AFAIK `UITextView` is a derivative of `UIScrollView` and thus likely has the same oddities in their SwiftUI counterparts. I haven't dug through the actual view hierarchy to see if this is necessarily the best (i.e. minimal) way to grab the `UIScrollView`

[^1]: XCode 13.1, iOS 15.0

### CODE REVIEW

**Comment by sapieneptus** [comment]
Also note, I'm not sure what, if any, changes should be applied on the MacOS side of things (assuming Apple may have made a similar change there as well)

File: Introspect/ViewExtensions.swift [hunk 1]
```diff
@@ -113,7 +113,7 @@ extension View {
 
     /// Finds a `UITextView` from a `SwiftUI.TextEditor`
     public func introspectTextView(customize: @escaping (UITextView) -> ()) -> some View {
-        introspect(selector: TargetViewSelector.siblingContaining, customize: customize)
+        introspect(selector: TargetViewSelector.siblingContainingOrAncestor, customize: customize)
```
Comment (sapieneptus) [inline Introspect/ViewExtensions.swift:116]:
```suggestion
        introspect(selector: TargetViewSelector.siblingContainingOrAncestorOrAncestorChild, customize: customize)
```

**Comment by X901** [comment]
Could you update your code with last changed on master ?
I have issue with "TextEditor" , your solution could help me, solve my [issue](https://github.com/siteline/SwiftUI-Introspect/issues/142)

**Comment by SplittyDev** [comment]
Hi, thanks a lot for the contribution! 

We haven't been able to update the CI to macOS 11 yet, since GitHub locked that worker down for a long time. It's now available and I'll try to update the CI workflow as soon as I find time to do so. That should fix the Simulator issues and allow for your tests to pass.

**Comment by davdroman** [comment]
Hi @sapieneptus. I've just revamped our CI in order to support testing on iOS 14, 15 and 16 simulators (#185). As it seems like the test for `TextEditor` is passing on all versions I'll be closing it for now to keep out backlog tidy, but feel free to resurface it if needed.

**Pull request closed without merging**